src/pages/HomePage/FeaturedProducts/FeaturedProducts.tsx

```diff
@@ -0,0 +1,81 @@
+import './FeaturedProducts.css'
+import chair1 from './chair1.png'
+import chair2 from './chair2.png'
+import chair3 from './chair3.png'
+import chair4 from './chair4.png'
+import group1 from './Group 141.png'
+import group2 from './Group 141 (1).png'
+import Rectangle1 from './Rectangle pink.png'
+import Rectangle2 from './Rectangle pinkk.png'
+import shopBasket from './shop.png'
+import favicon from './uil_heart-alt.png'
+import searchicon from './uil_search-plus.png'
+
+export const FeaturedProducts = () => {
+    return(
+    <div className="FeaturedProducts">
+      <h1>Featured Products</h1>
+      <div className="chairs">
+        <div className="chairBox">
```

Comment:
seems like you've got to create a component with name FeaturedProduct and move html for "<div className="chairBox">" to this component 
then you have to created an array of featured products in the FeaturedProducts component and use featuredProducts.map(...) to render your "FeaturedProduct" components